src/views/document/utils/field-viewers/FieldView.vue

```diff
@@ -47,7 +47,7 @@ export default {
         return false;
       }
 
-      if (Array.isArray(value) && value.length === 0) {
+      if (Array.isArray(value) && !value.length) {
```

Comment:
previous version seems better

Reply:
I totally disagree 🙂 
Ok, it's mainly a matter of taste. But there are also more subtle reasons for this IMHO, among which the fact that any falsy value will be tested (think what if `length` was `undefined`? OK again, here we test for array first)

And you save 5 characters, it's like saving baby seals 👍

Reply:
Brigitte Bardot approves